ceph_installer/util.py

```diff
@@ -38,6 +38,38 @@ def generate_inventory_file(inventory, task_uuid, tmp_dir=None):
     return inventory_file.name
 
 
+def parse_monitors(monitors):
+    """
+    Given a list of dictionaries, returns a list of hosts that
+    can be used in an ansible inventory. These host lines can include
+    host variables as well.
+
+    For example, monitors in this format:
+
+        [
+            {"host": "mon0.host", "interface": "eth0"},
+            {"host": "mon1.host", "interface": "enp0s8"},
+        ]
+
+    Would return the following:
+
+        ["mon0.host monitor_interface=eth0", "mon1.host monitor_interface=enp0s8"]
+    """
+    hosts = []
+    var_map = dict(
+        interface="monitor_interface",
+    )
+    for mon in monitors:
+        host = []
+        host.append(mon["host"])
+        for k, v in var_map.iteritems():
```

Comment:
Yeah, that was the thought anyway. I can't think of one off the top of my head at the moment though.